apps/modernization-ui/src/apps/search/layout/result/header/options/list/SearchResultsListOptions.tsx

```diff
@@ -27,55 +28,34 @@ const SearchResultsListOptions = ({ disabled = false }: Props) => {
         }
     }, []);
 
+    const isActive = (field: SortField, dir: Direction) => property === field && direction === dir;
+
+    const renderSortButton = (field: SortField, dir: Direction, label: string) => (
+        <Button
+            className={classNames(styles.optionItem, isActive(field, dir) ? styles.active : '')}
```

Comment:
The `classNames` function is made to handle situations like this where you are manipulating strings to set which css classes are applied.

```
className={classNames(styles.optionItem, {[styles.active] : isActive(field, dir) })}
```

This will only add the classes if the condition i met

Reply:
good point, lesson learned.